### PR TITLE
Fix bug causing errors on submission api endpoint [OSF-6942]

### PR DESCRIPTION
**Pull request by jnayak1**
None of the submission detail methods work on the API currently. This is because we pass in submission_id to the view in urls.py, but we're trying to use pk in the view. This PR fixes this by just using args, kwargs, and the superclass' method.

### CODE REVIEW

File: meetings/submissions/views.py [hunk 1]
```diff
@@ -42,18 +42,18 @@ class SubmissionViewSet(viewsets.ModelViewSet):
     #  OSF's node url
     node_url = '{}v2/nodes/'.format(settings.OSF_API_URL)
 
-    def retrieve(self, request, pk=None):
+    def retrieve(self, request, submission_id=None):
```
Comment (Ghalko) [inline meetings/submissions/views.py]: If we can we should use `self, request, *args, **kwargs`

**Comment by Tav0** [comment]
Still not sure if this is a bug. I will let others review because everything it is working for me.

**Comment by jnayak1** [comment]
image

**Comment by jnayak1** [comment]
This was also addressed in https://github.com/CenterForOpenScience/osf-meetings/pull/83. I will close this PR in favor of that one.

**Pull request closed without merging**